### PR TITLE
Logo Changed

### PR DESCRIPTION
**Pull request by aanchalrakheja**
## What is the change?
Created a new logo for the website

## Related issue?
close: #746 

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Did you lint your code before making the Pull Request?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:
image
image
image

### CODE REVIEW

File: public/css/styles.css [hunk 1]
```diff
@@ -70,18 +70,32 @@ body {
 	backdrop-filter: blur(10px);
 }
 /* -------------------------------------------------- */
-
+#logo{
+	margin-top:-15px;
+	width:12rem;
+	height:7rem;
+	/* max-height:13rem;
+	max-width:25rem; */
```
Comment (RushanKhan1) [inline public/css/styles.css:78]: Please remove the unnecessary comments. Else everything looks good to me. 👍🏻

**Review by RushanKhan1 — CHANGES_REQUESTED** [review]

File: public/css/styles.css (hunk 1 above)
Comment (aanchalrakheja) [reply public/css/styles.css:78]: okay

**Comment by aanchalrakheja** [comment]
Please check once

**Comment by Pranav016** [comment]
@aanchalrakheja the logo looks really good on the website, thanks!

**Pull request merged**